### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by MaxLeiter** [MaxLeiter/Drift#10]

**Comment by kjaonline** [comment]
Hey I want to contribute. Where do I start? Do you have a discord?

**Comment by MaxLeiter** [comment]
Hey @kjaonline! We have an IRC channel which you can find some more info on in the README. If you aren't familiar with IRC, I recommend checking https://libera.chat/guides/clients for a guide to selecting a client.

Regarding the documentation, I did a fair amount of it today, but I can think on some tasks and get back to you tomorrow with details.

**Comment by jazcarate** [comment]
Dunno how useful it is as a data point, but: as a complete stranger to the project, the README.md was more than enough to setup my dev environment 🤷

**Comment by MaxLeiter** [comment]
Yeah I’ve been improving it incrementally, but I’m leaving this up as a general call for installation guides and the like
